lib/LocalVerifiableCredentialStore.js

```diff
@@ -118,11 +140,12 @@ export class LocalVerifiableCredentialStore {
   async delete({id}) {
     try {
       const selector = {
-        'content.id': id,
         'meta.profileId': this.profileId,
+        'content.id': id
```

Comment:
This is fine -- but I'm just curious -- does order matter?

Reply:
I don't think order matters in the selector but I was just trying to match the order with how it was ordered in the index.